src/native/eventpipe/ep-session.h

```diff
@@ -13,6 +13,15 @@
 #endif
 #include "ep-getter-setter.h"
 
+#ifdef HAVE_LINUX_USER_EVENTS_H
+#include <linux/user_events.h> // DIAG_IOCSREG
+#include <sys/ioctl.h> // event_reg
+#endif // HAVE_LINUX_USER_EVENTS_H
+
+
+#ifdef HAVE_SYS_UIO_H
+#include <sys/uio.h> // iovec
+#endif // HAVE_SYS_UIO_H
```

Comment:
Changing from `#ifdef` to `#if`. Missed that it was `#cmakedefine01` + `#ifdef`, which would always result in the macro being defined.

Opting for `#cmakedefine01` and `#if`